src/commands/common/send-telemetry-command.js

```diff
@@ -11,45 +10,41 @@ class SendTelemetryCommand extends Command {
         super(ctx);
         this.logger = ctx.logger;
         this.config = ctx.config;
-        this.telemetryInjectionService = ctx.telemetryInjectionService;
         this.networkModuleManager = ctx.networkModuleManager;
         this.blockchainModuleManager = ctx.blockchainModuleManager;
+        this.repositoryModuleManager = ctx.repositoryModuleManager;
+        this.telemetryModuleManager = ctx.telemetryModuleManager;
     }
 
     /**
      * Performs code update by fetching new code from github repo
      * @param command
      */
     async execute() {
-        if (!this.config.telemetry.enabled || !this.config.telemetry.sendTelemetryData) {
+        const implementationName = this.telemetryModuleManager.getImplementationNames();
+        if (
+            !this.config.modules.telemetry.enabled ||
+            !this.config.modules.telemetry.implementation[implementationName].config
```

Comment:
you can call getModuleConfiguration method from telemetryModuleManager without attributes to fetch configuration, not like this